### PR TITLE
[14.0][MIG] sale_quotation_number: Migration to 14.0

### PR DESCRIPTION
**Pull request by Vanneri**
Migration to 14.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
That's it, thanks! Unfortunately, I'm now seeing that another PR is already opened in #1339 :cry:

**Comment by Vanneri** [comment]
> That's it, thanks! Unfortunately, I'm now seeing that another PR is already opened in #1339

oh ok , then i will close this one.
Please let me know.

**Comment by Vanneri** [comment]
@kittiu should i close or can you consider this request? 
@pedrobaeza

**Comment by kittiu** [comment]
> @kittiu should i close or can you consider this request?
> @pedrobaeza

Let's keep this one, as #1339 is WIP.
In the future, may be it is a good idea to search for module name in open PR before start, just to be sure. :)

**Review by kittiu — APPROVED** [review]
Code reviewed.

File: sale_quotation_number/models/res_company.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    keep_name_so = fields.Boolean(
+        string="Use Same Enumeration",
+        help="If this is unchecked, quotations use a different sequence from "
```
Comment (newtratip) [inline sale_quotation_number/models/res_company.py:15]:
```suggestion
        help="If this is unchecked, quotations use a different sequence from sale orders",
```

File: sale_quotation_number/models/res_company.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "sale orders",
```
Comment (newtratip) [inline sale_quotation_number/models/res_company.py:16]:
```suggestion
        
```

Comment (Vanneri) [reply sale_quotation_number/models/res_company.py:16]: committed as per the review

**Comment by Vanneri** [comment]
@kittiu is it ok to proceed?
@newtratip I changed as per you reviewed but codecov is getting failed so i reverted the commit.

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Review by newtratip — APPROVED** [review]
> @newtratip I changed as per you reviewed but codecov is getting failed so i reverted the commit.

I'm OK.

**Comment by Vanneri** [comment]
@dreispt please retry the merge if possible

**Pull request merged**